src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
```

Comment:
The `split` method here will create an array with empty strings for the consecutive semicolons. It's good that you're using `filter` to remove empty entries, but consider that entries with only whitespace characters will also pass this check. You might want to filter out entries that only contain whitespace as well.